src/custom_init.cr

```diff
@@ -13,14 +13,18 @@ class LuckyCli::CustomInit < LuckyCli::Init
   private def generate_project(project_name)
     api_only = false
     authentication = true
+    project_directory = "."
     OptionParser.parse do |parser|
       parser.banner = <<-TEXT
       Usage: lucky custom.init NAME [arguments]
 
-      Example: lucky custom.init my_api --api --no-auth
+      Example: lucky custom.init my_api --api --no-auth --dir ~/Projects/
```

Comment:
Maybe add a note that all these arguments are optional and all it requires is the project name

Reply:
How about 2 examples. 1 that shows with just the name, and a 2nd with the options?

Reply:
Yeah that sounds good!